src/components/CurriculumComponents/CurriculumHeaderTabNav/index.tsx

```diff
@@ -37,7 +37,6 @@ const CurriculumHeaderTabNav = ({
           size="large"
           variant={variant}
           aria-current={link.isCurrent ? "page" : undefined}
-          aria-controls={link.page}
```

Comment:
Yeah it was breaking pa11y unfortunately, so had to change it.

Reply:
Further context: this `aria-controls` property doesn't actually do anything, because this is just a link in a nav and the link has a `href` property that tells the user/browser/assistive technology where it goes. Also you can't really 'control' a page region because the region itself isn't shown-and-hidden on demand, it's a route change.